sites/fast-website/src/app/components/card-section/index.ts

```diff
@@ -0,0 +1,14 @@
+import { customElement, FASTElement } from "@microsoft/fast-element";
```

Comment:
Your other names start capitalized, but, not "customElement" or "css", "display" etc.  What naming standard are we following?  Can we align these to be consistent, throughout this PR and actually entire project?

Reply:
FYI, this is the API of `fast-element` and intentional. @EisenbergEffect can provide more detail but we're mapping to standards here.

Reply:
customElement is the function export defined in fast-element.  css and display are functions as well, so the casing seems to follow convention.

Reply:
Currently, decorators are functions, so we apply function casing. `customElement` is one such decorator. `html` and `css` are tag template functions, so we also apply function casing. `FASTElement` is a class, so we apply class casing.

Reply:
@EisenbergEffect Can this be added to the documentation if not already?